### DISCUSSION TITLE
Update nspanel_pro_tools_apk while keeping settings

### DISCUSSION THREAD

**Discussion by acseven** [seaky/nspanel_pro_tools_apk#99] [General]
Hi, I've just updated to v2.2.0 (thanks!!) by uninstalling v2.1.0 and after installing v2.2.0. However, all my previous configurations were reset, and it's a bummer having to setup all of it again. 

Is there a way to update the package while not losing the current settings?

**Comment by seaky** [comment]
Hi,

yes of course, just install as the manual describes!
```
adb install -r <apk>

```

**Reply by acseven** [reply]
Thank you, must’ve read install/uninstall somewhere without checking for better information.
